src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift

```diff
@@ -127,6 +127,16 @@ final class ExposureDetectionViewController: DynamicTableViewController, Require
 		return cell
 	}
 
+	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```

Comment:
All the other cells are using the .action property of DynamicCell. Cant it be used for the link cell as well?
If not, I think you need to call super here, otherwise maybe the other actions won't work.

Reply:
This 'link in text' stuff is according to the design not a normal action. Normally Links are opened right from a UITextView and without any extra code. However, UILabels don't support links in attributed texts so I created this one for this one-off abomination.

I'll check actions but if that turns out to require tons of extra hacks I'd rather go with this and add that missing `super` call.

Reply:
I'm not talking about the 'link in text' stuff.
The DynamicTableViewController is already implementing "didSelectRowAt". It calls the corresponding .action of the dynamic cell at the index path.

Reply:
It's using actions now, please review.